.github/workflows/staging-deploy.yaml

```diff
@@ -0,0 +1,48 @@
+name: Deploy to staging
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - ".happy/**"
+      - "api_server/**"
+
+# https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy-to-staging:
```

Comment:
What do you think about trying to switch over ARM for the api? I already added a `platform: linux/arm64` line to the docker-compose.yml. So we either need to remove that, or add a `runs_on` parameter to this job to build on ARM, like so:

```
    runs-on: [ARM64, self-hosted, Linux]
```